### PR TITLE
quick fix for delete.type #4315 for next steps

### PR DESCRIPTION
**Pull request by sixfourtwelve**
## Overview

More of a temp solution to negate the issue. See https://github.com/unisonweb/unison/issues/4315 for the bigger/fuller fix

## Test coverage

Manual testing

### CODE REVIEW

**Comment by sixfourtwelve** [comment]
old pr for reference: https://github.com/unisonweb/unison/pull/4316

File: unison-cli/src/Unison/CommandLine/InputPatterns.hs [hunk 1]
```diff
@@ -727,22 +727,22 @@ deleteGen suffix target mkTarget =
         )
 
 delete :: InputPattern
-delete = deleteGen Nothing "term or type" (DeleteTarget'TermOrType DeleteOutput'NoDiff)
+delete = deleteGen Nothing exactDefinitionTermQueryArg "term or type" (DeleteTarget'TermOrType DeleteOutput'NoDiff)
```
Comment (aryairani) [inline unison-cli/src/Unison/CommandLine/InputPatterns.hs]: This one maybe needs a TermOrTypeQueryArg

**Review by aryairani — APPROVED** [review]

**Comment by aryairani** [comment]
You'll need to `stack exec transcripts` and then check in the results

File: unison-cli/src/Unison/CommandLine/InputPatterns.hs (hunk 1 above)
Comment (sixfourtwelve) [reply unison-cli/src/Unison/CommandLine/InputPatterns.hs]: Updated!

**Review by ChrisPenner — APPROVED** [review]

**Pull request merged**